xml/Microsoft.JScript/ActivationObject.xml

```diff
@@ -323,7 +323,8 @@
       <MemberSignature Language="DocId" Value="M:Microsoft.JScript.ActivationObject.GetMemberValue(System.String,System.Int32)" />
       <MemberSignature Language="VB.NET" Value="Public Function GetMemberValue (name As String, lexlevel As Integer) As Object" />
       <MemberSignature Language="C++ CLI" Value="public:&#xA; virtual System::Object ^ GetMemberValue(System::String ^ name, int lexlevel);" />
-      <MemberSignature Language="F#" Value="abstract member GetMemberValue : string * int -&gt; obj&#xA;override this.GetMemberValue : string * int -&gt; obj" Usage="activationObject.GetMemberValue (name, lexlevel)" />
+      <MemberSignature Language="F#" Value="override this.GetMemberValue : string * int -&gt; obj" Usage="activationObject.GetMemberValue (name, lexlevel)" FrameworkAlternate="netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5" />
```

Comment:
why only F# has different signatures for different versions?

Reply:
well, in this case it's pretty much exactly the same as the other issue ... the method is reporting as `virtual sealed`; so that's why the C# and other formatters omit that part of the signature (the F# formatter just doesn't do that, hence the difference).

However, in this case, the difference here is that the type is abstract ... offhand, I'm not really sure how a method like that would be defined; however it's undeniable that the metadata is reporting both `IsVirtual==true` and `IsFinal==true`. So for all intents and purposes this is working as implemented.